scipy/_lib/_ccallback_c.pxd

```diff
@@ -1,3 +1,5 @@
+# cython: language_level=3
```

Comment:
Is it possible to avoid adding this to every pyx file individually by changing our build infrastructure in one place (or maybe just in each submodule)? For example if we can set the Cython command line options in setup.cfg or in setup.py files you can change the language level that way.

Reply:
As far as I know, these directives cannot be supplied on the command line. Unfortunately, because it took quite a bit of time to add it to every .pyx and .pxd file, we've got a lot of those.

Reply:
In [the docs](https://cython.readthedocs.io/en/latest/src/userguide/source_files_and_compilation.html#compilation) it says

    language_level (2/3/3str)
        Globally set the Python language level to be used for module compilation.
        Default is compatibility with Python 2. To enable Python 3 source code
        semantics, set this to 3 (or 3str) at the start of a module or pass the
        “-3” or “–3str” command line options to the compiler...

So the question is, is there a place we can supply command line options to `cythonize` (or related functions) that would work?

Reply:
(And similarly the `cythonize` and related functions have a `language` option, so if we don't use the command line compilation, then maybe there is a simple and elegant way to have a few-line wrapper for Cython build infrastructure that just passes this `language='3'` var. This is a build infrastructure issue that I don't know enough to answer, but @rgommers if you also don't know offhand, I can look into it if you don't want to / can't but think it would be worthwhile.)

Reply:
Maybe here adding `-3` or `-3str` would work?

https://github.com/scipy/scipy/blob/master/tools/cythonize.py#L87

Reply:
damn that does work. somehow I missed that in the docs. updated. thanks Eric